collector/src/execute.rs

```diff
@@ -894,6 +900,44 @@ impl<'a> Processor for BenchProcessor<'a> {
     }
 }
 
+fn store_documentation_size_into_stats(stats: &mut Stats, doc_dir: &Path) {
+    match get_file_count_and_size(doc_dir) {
+        Ok((count, size)) => {
+            stats.insert("size:doc_files".to_string(), count as f64);
```

Comment:
I will say that I still feel like `size:doc_files_count` would be worth the long name... But this is not a blocking concern.

Reply:
Ok, I have renamed it :)